glean-core/src/error.rs

```diff
@@ -115,3 +121,11 @@ impl From<Error> for ExternError {
         ffi_support::ExternError::new_error(ffi_support::ErrorCode::new(42), format!("{}", error))
     }
 }
+
+impl From<serde_json::error::Error> for Error {
```

Comment:
Fun fact: we should never even run into that as we're building our content from JsonValue and kinda guarantee that it maps to valid json. Anyway, the code requires it.

Reply:
Yeah -- theoretically `to_string` returns one, but it seems unlikely that serialization would fail.  Anyway, this lets us at least not have an `unwrap` hanging there looking suspicious.